spec/views/posts/show.html.erb_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe 'Post show', type: :feature do
+  before :each do
+    @user = User.new(name: 'Santiago', photo: 'http://photo.com', bio: 'This is me.', posts_counter: 5,
+                     email: 'santiagovelosa@gmail.com', password: '123456')
+    @user.skip_confirmation!
+    @user.save!
+    @post = @user.posts.create(title: 'Post #3', text: 'This is a new text', comments_counter: 5, likes_counter: 5)
+    @comment = @post.comments.create(author: @user, text: 'Hello, how are you?')
+    visit new_user_session_path
+    fill_in 'Email', with: 'santiagovelosa@gmail.com'
+    fill_in 'Password', with: '123456'
+    click_button 'Log in'
+    visit user_post_path(@user, @post)
+  end
+
+  it 'show post information' do
+    expect(page).to have_content('Post #3')
+    expect(page).to have_content('Santiago')
+    expect(page).to have_content('comments: 1')
+    expect(page).to have_content('Likes: 5')
+    expect(page).to have_content('This is a new text')
+    expect(page).to have_selector('.comment-details', count: 1)
+    expect(page).to have_content('Hello, how are you?')
+  end
```

Comment:
- [ ]  You should add a test that asserts you can see the username of each commenter. [Requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Integration_specs_project.md#:~:text=I%20can%20see%20the%20username%20of%20each%20commentor.)

Reply:
Hi @ahmetbozaci,
Actually, we implemented it you can see in line 20.
![0ac65fda-5740-4cda-845e-7c940412c981](https://user-images.githubusercontent.com/68796232/162241385-5eb5bf3f-8c4a-40ba-9cf9-64799f879f51.jpg)

Reply:
Yes, you are right. This [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Integration_specs_project.md#:~:text=I%20can%20see%20who%20wrote%20the%20post.) is same.

Reply:
Ok we added it now.